wezterm-gui/src/main.rs

```diff
@@ -886,6 +887,7 @@ pub fn run_ls_fonts(config: config::ConfigHandle, cmd: &LsFontsCommand) -> anyho
     let unicode_version = UnicodeVersion {
         version: config.unicode_version,
         ambiguous_are_wide: config.treat_east_asian_ambiguous_width_as_wide,
+        cellwidths: setcellwidths(config.cellwidths.clone()),
```

Comment:
I think it's worth adding a helper method to the `Config` class that encapsulates this:

in `config/src/config.rs`:

```rust
impl Config {
   pub fn unicode_verson(&self) -> UnicodeVersion {
      ...
   }
}
```

then the logic in this file is simply:

```rust
let unicode_version = config.unicode_version();
```

Reply:
added unicode_version() in struct ConfigHandle